### PR TITLE
F#: split client and runtime

### PR DESCRIPTION
**Pull request by aprooks**
issue: #112 

only folder split. Nuget of runtime was not checked

### CODE REVIEW

**Comment by aprooks** [comment]
@yevhen should be ready for merge;)

File: Build/Orleankka.FSharp.Runtime.nuspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Orleankka.Fsharp.Runtime</id>
```
Comment (yevhen) [inline Build/Orleankka.FSharp.Runtime.nuspec]: FS^harp

**Comment by aprooks** [comment]
thanks I will fix tomorrow

**Comment by yevhen** [comment]
@aprooks what you have set on your machine for core.autocrlf?

**Comment by aprooks** [comment]
`core.autocrlf=false`

**Comment by aprooks** [comment]
`core.autocrlf=false`

**Comment by yevhen** [comment]
Ok, that explains crlf issue. Could you pls set it to true for Orleankka repo?

**Comment by aprooks** [comment]
Sure. Line endings handling is still a mystery for me :(

**Comment by yevhen** [comment]
Great work! 👍

**Pull request merged**